keylime/keylime_agent.py

```diff
@@ -279,9 +279,8 @@ def do_POST(self):
             shutil.rmtree("%s/unzipped" % secdir)
 
         # write out key file
-        f = open(secdir + "/" + self.server.enc_keyname, 'w', encoding="utf-8")
-        f.write(base64.b64encode(self.server.K).decode())
-        f.close()
+        with open(secdir + "/" + self.server.enc_keyname, 'w', encoding="utf-8") as f:
```

Comment:
Can you fix the string formatting to `os.path.join`?.

I know that this is not directly related to this issue, but I try to convert them if I change the line anyway.

Reply:
Fixed locally.